src/common/middleware/call-api.js

```diff
@@ -5,7 +5,7 @@ export const CALL_API = 'CALL_API';
 
```

Comment:
I've rewritten the call-api middleware to use promises instead of async/await. Source mapping for async/await code seems to be a bit flakey, I found it hard to step through this code to debug error responses handled by Fetch and problems in reducers.

I think this is a reasonable tradeoff within the scope of the middleware, I would still be happy to use async/await elsewhere in the application.

It is definitely something we should keep an eye on.